assets/src/dashboard/app/serializers/storyPreview.js

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export default function reshapeStoryPreview(storyProps = {}) {
+  const story = {
+    ...storyProps,
+    date: storyProps.date || '',
+    title: storyProps.title,
+  };
+  const pages = [...storyProps.pages];
+  const metadata = {
+    publisher: {
+      name: '',
+    },
+    fallbackPoster: '',
+  };
```

Comment:
Would appreciate eyes on this if there's a better way to do this reshape - from playing around with this feature this set of props seems to be these minimal necessary to get a working preview without console errors.

Reply:
This works for me as is. Every other way I can think to handle this isn't much simpler or more terse